lib/std/jsfetch.nim

```diff
@@ -42,56 +43,56 @@ type
     frpOriginWhenCrossOrigin = "origin-when-cross-origin"
     frpUnsafeUrl = "unsafe-url"
 
-  Body* = ref object of JsRoot  ## https://developer.mozilla.org/en-US/docs/Web/API/Body
-    bodyUsed*: bool
```

Comment:
`bodyUsed*: bool` still exists in the API and should not be removed:

- https://developer.mozilla.org/en-US/docs/Web/API/Request/bodyUsed
- https://developer.mozilla.org/en-US/docs/Web/API/Response/bodyUsed

Reply:
As you linked, it is still there in `Request` and `Response`, just not in the form of `Body`.
See L47 and L55

```
  Response* = ref object of JsRoot  ## https://developer.mozilla.org/en-US/docs/Web/API/Response
    bodyUsed*, ok*, redirected*: bool
    typ* {.importjs: "type".}: cstring
    url*, statusText*: cstring
    status*: cint
    headers*: Headers
    body*: cstring

  Request* = ref object of JsRoot  ## https://developer.mozilla.org/en-US/docs/Web/API/Request
    bodyUsed*, ok*, redirected*: bool
    typ* {.importjs: "type".}: cstring
    url*, statusText*: cstring
    status*: cint
    headers*: Headers
    body*: cstring
```